docs/src/examples/collections/Table/Variations/TableExampleSortable.js

```diff
@@ -1,62 +1,51 @@
 import _ from 'lodash'
-import React, { Component } from 'react'
-import { Table } from 'semantic-ui-react'
+import React, { useState } from "react";
```

Comment:
```suggestion
import React from "react";
```

I suggest to use `React.useState`/`React.useReducer` as it gives more context from where hooks came.